packages/react-reconciler/src/__tests__/ReactBlocks-test.js

```diff
@@ -1,351 +0,0 @@
-/**
- * Copyright (c) Facebook, Inc. and its affiliates.
- *
- * This source code is licensed under the MIT license found in the
- * LICENSE file in the root directory of this source tree.
- *
- * @emails react-core
- * @jest-environment node
- */
-
-let React;
-let ReactNoop;
-let useState;
-let Suspense;
-let block;
-let readString;
-let resolvePromises;
-let Scheduler;
-
-describe('ReactBlocks', () => {
-  beforeEach(() => {
-    jest.resetModules();
-
-    Scheduler = require('scheduler');
-    React = require('react');
-    ReactNoop = require('react-noop-renderer');
-
-    block = React.unstable_block;
-    useState = React.useState;
-    Suspense = React.Suspense;
-    const cache = new Map();
-    let unresolved = [];
-    readString = function(text) {
-      let entry = cache.get(text);
-      if (!entry) {
-        entry = {
-          promise: new Promise(resolve => {
-            unresolved.push(() => {
-              entry.resolved = true;
-              resolve();
-            });
-          }),
-          resolved: false,
-        };
-        cache.set(text, entry);
-      }
-      if (!entry.resolved) {
-        throw entry.promise;
-      }
-      return text;
-    };
-
-    resolvePromises = () => {
-      const res = unresolved;
-      unresolved = [];
-      res.forEach(r => r());
-    };
-  });
-
-  // @gate experimental
-  it('renders a simple component', () => {
```

Comment:
Nothing worth converting here?

Reply:
I don't think so. These are all just "client blocks" which is just components.